doc/src/fix_wall_gran.rst

```diff
@@ -175,8 +177,6 @@ the clockwise direction for *vshear* > 0 or counter-clockwise for
 *vshear* < 0.  In this case, *vshear* is the tangential velocity of
 the wall at whatever *radius* has been defined.
 
-
```

Comment:
why delete these lines?

Reply:
Oops, that was a mistake. I'll revert the deletion.